qa/L0_shared_memory/shared_memory_test.py

```diff
@@ -292,7 +328,7 @@ def test_too_big_shm(self):
     def test_mixed_raw_shm(self):
         # Mix of shared memory and RAW inputs
         error_msg = []
-        shm_handles = self._configure_sever(shm_byte_size=256)
+        shm_handles = self._configure_server()
```

Comment:
Was this supposed to have a value of 256 passed to it?

Reply:
I think it was only needed because I had a bug so I removed it..